crates/forge/src/main.rs

```diff
@@ -98,6 +98,10 @@ struct TestArgs {
     /// Run tests that failed during the last run
     #[arg(long)]
     rerun_failed: bool,
+
+    /// Save call traces of all tests
```

Comment:
All tests or only non-fuzzing passed ones?

Reply:
I forgot to update it 👍